### PR TITLE
zippyshare direct link generator

### PR DESCRIPTION
**Pull request by aryanvikash**

### CODE REVIEW

File: plugins/zippyshare.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+@userge.on_cmd("zippy", about={
+    'description': "generate Direct link of zippyshare url",
+    'usage': "{tr}zippy : [Zippyshare Link ]",
+    'examples': "{tr}zippy https://www10.zippyshare.com/v/dyh988sh/file.html"}, del_pre=True)
+async def zippyshare(m: Message):
+    url = m.filtered_input_str
+    loop = asyncio.get_event_loop()
+    await m.edit("Generating url ....")
+    try:
+        direct_url, fname = await loop.run_in_executor(None, generate_zippylink, url)
```
Comment (rking32) [inline plugins/zippyshare.py:28]: great, can you use pool instead of this

Comment (aryanvikash) [reply plugins/zippyshare.py:28]: Don't know about pool .. will you please Do it :)

Comment (rking32) [reply plugins/zippyshare.py:28]: ok, thanks for this plugin

Comment (aryanvikash) [reply plugins/zippyshare.py:28]: Don't know about pool . will you please do it :)

**Comment by rking32** [comment]
@aryanvikash `decrypt_dlc` and `get_file` functions not using ?

**Comment by aryanvikash** [comment]
> @aryanvikash `decrypt_dlc` and `get_file` functions not using ?

NO  !  I am such  irresponsible . Please remove it .

**Comment by rking32** [comment]
@aryanvikash ok, i have commented that. so we can use it if we want in future :+1:

**Pull request merged**